resources/assets/sass/people.scss

```
@@ -628,6 +628,7 @@
       a.edit {
         position: relative;
         top: 1px;
+        margin-right: 5px;
```

Carefull about the orientation here: RTL langages might need a `margin-left` (to be verified)
```suggestion
    @if $htmldir == ltr {
        margin-right: 5px;
    } @else {
        margin-left: 5px;
    }
```

---

Good question! I don't know yet.
Tip: you can test by adding your langage in (this list )[https://github.com/monicahq/monica/blob/master/app/Helpers/LocaleHelper.php#L72] temporarly.

---

I think this makes sense.. still need to test it though